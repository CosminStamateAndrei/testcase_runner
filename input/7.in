3 2 3
-9 4 
7 3 
-5 3 
-1 5 -10 
-2 -1 -5 
