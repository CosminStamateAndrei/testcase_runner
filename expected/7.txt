Matrix A
  -9    4 
   7    3 
  -5    3 
Matrix B
  -1    5  -10 
  -2   -1   -5 
Matrix C
   1  -49   70 
 -13   32  -85 
  -1  -28   35 
